src/domains/checkout-app/04_apim_checkout_authentication.tf

```
@@ -26,7 +26,7 @@ module "apim_checkout_authentication" {
 }
 
 resource "azurerm_api_management_api_version_set" "checkout_auth_service_api_v1" {
-  name                = "${local.parent_project}-auth-service-api"
+  name                = "${var.prefix}-${var.env_short}-${var.domain}-auth-service-api"
```

```suggestion
  name                = "${local.project}-auth-service-api"
```

Could you use the [local project value](https://github.com/pagopa/pagopa-infra/blob/396f5062ddea6ab9d70e2d7ac7ac0f7c1d3703d4/src/domains/checkout-app/99_locals.tf#L2) here, like it's done for other domains such eCommerce?

---

for the apim API, it is not a good thing to add the location (weu/itn included in local.project) because we risk creating the bug as well as for the wallet API, which are called `pagopa-p-itn-pay-wallet-payment-wallet-for-ecommerce-api-v1`. in fact all the API are in weu but in the name of the bee groups it is better not to consider the location

---

Ok for not include location in api name, but instead of duplicating string interpolation for all api we could create a new local value that interpolates prefix, env_short and domain and use it

---

Fixed in [82789868bece135368009fadbd7daa6046c7c3de](https://github.com/pagopa/pagopa-infra/pull/2806/commits/82789868bece135368009fadbd7daa6046c7c3de)